src/Features/CSharp/Portable/SignatureHelp/AbstractCSharpSignatureHelpProvider.cs

```diff
@@ -53,7 +53,7 @@ protected static SymbolDisplayPart NewLine()
                 parameter.Name,
                 parameter.IsOptional,
                 parameter.GetDocumentationPartsFactory(semanticModel, position, formatter),
-                parameter.ToMinimalDisplayParts(semanticModel, position));
+                parameter.ToMinimalDisplayParts(semanticModel, position, SymbolDisplayFormat.MinimallyQualifiedFormat.AddMiscellaneousOptions(SymbolDisplayMiscellaneousOptions.AllowDefaultLiteral)));
```

Comment:
can you make this a static variable so we don't have to allocate a fresh instance on each param?  thanks :)